tests-rtl/src/store.js

```diff
@@ -0,0 +1,21 @@
+import { initialState as schemaInitialState } from "./api";
+import { LogContext, each, log, parallel, take } from "starfx";
+import { configureStore } from "starfx/store";
+
+export function setupStore({ initialState = {} }) {
+  const store = configureStore({
+    initialState: {
+      ...schemaInitialState,
+      ...initialState,
+    },
+  });
+
+  const tsks = [];
```

Comment:
This needs `api.bootup`

Reply:
Resolved in ffadf58cd9f4c2f30480e8bff8af2db8ce8ad259.